src/insert.rs

```diff
@@ -95,24 +95,21 @@ impl InsertState {
         }
     }
 
+    fn expect_client_mut(&mut self) -> &mut Client {
+        let Self::NotStarted { client, .. } = self else {
+            panic!("cannot modify client options while an insert is in-progress")
+        };
+
+        client
+    }
```

Comment:
Does it make sense to mark it as inline?
Also, perhaps better to name it `expect_not_started` instead?

Reply:
Given that none of these others are inlined? I don't know. It's not called in a super hot path.

The naming sort of matches `client_with_sql`. I originally was going to call it `client_mut()` but every time it'd be called it would be followed by `.expect()` anyway so this made the most sense to me.